pkgs/tools/misc/logstash/7.x.nix

```diff
@@ -3,12 +3,13 @@
 , lib, stdenv
 , fetchurl
 , makeWrapper
+, nixosTests
 , jre
 }:
 
 with lib;
 
-stdenv.mkDerivation rec {
+let this = stdenv.mkDerivation rec {
```

Comment:
I am not sure why we need the let in here. Do I miss something?

Reply:
It's for the assertion in `passthru.tests`.